rc/vue.kak

```diff
@@ -0,0 +1,88 @@
+
+# http://w3.org/html
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*\.vue %{
+    set-option buffer filetype vue
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=vue %{
+    require-module vue
+
+#     hook window ModeChange pop:insert:.* -group "%val{hook_param_capture_1}-trim-indent"  html-trim-indent
+#     hook window InsertChar '>' -group "%val{hook_param_capture_1}-indent" html-indent-on-greater-than
+#     hook window InsertChar \n -group "%val{hook_param_capture_1}-indent" html-indent-on-new-line
+# 
+    # hook -once -always window WinSetOption "filetype=.*" "
+        # remove-hooks window ""%val{hook_param_capture_1}-.+""
+    # "
+}
+
+hook -group html-highlight global WinSetOption filetype=vue %{
+#    add-highlighter "window/%val{hook_param_capture_1}" ref html
+#    hook -once -always window WinSetOption "filetype=.*" "
+#        remove-highlighter ""window/%val{hook_param_capture_1}""
+#    "
+}
+
+
+provide-module vue %[
+
+try %{
+    require-module css
+    require-module javascript
+}
+
+# Highlighters
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+add-highlighter shared/html regions
+
+hook global BufCreate .*\.xml %{
+    set-option buffer filetype xml
+}
+add-highlighter shared/html/comment region <!--     -->                  fill comment
```

Comment:
If you want to depend on the html module, you can use
`add-highlighter shared/vue/comment ref html/comment`
and so on. Hard dependencies between modules are not conventional, but for this one it's probably okay.